ee/server/service/calendar.go

```diff
@@ -214,7 +215,8 @@ func (svc *Service) processCalendarEvent(ctx context.Context, eventDetails *flee
 			return "", false, err
 		}
 
-		return calendar.GenerateCalendarEventBody(ctx, svc.ds, team.Name, host, &sync.Map{}, conflict, svc.logger), true, nil
+		body, generatedTag = calendar.GenerateCalendarEventBody(ctx, svc.ds, team.Name, host, &sync.Map{}, conflict, svc.logger)
```

Comment:
What are the advantages of assigning the value of `generatedTag` like this instead of as another return value from `genBodyFn`?

Reply:
This tag is only used/needed by the cron job and callback. It is not needed by the code that's interacting with the Google API.